spec/graphql/query/literal_input_spec.rb

```diff
@@ -35,40 +35,56 @@
           end
         end
 
+        query = GraphQL::ObjectType.define do
+          name "Query"
+
+          field :top, type, resolve: ->(_, _, _) { true }
```

Comment:
I moved the fields into this useless looking "top" field so that the path info / results could be shown better (before the whole query would be nil and the errors would have no path info).